projects/plugins/boost/app/assets/src/js/features/error-notice/error-notice.module.scss

```diff
@@ -0,0 +1,73 @@
+@import "../../../css/main/mixins.scss";
+@import "../../../css/main/variables.scss";
+
+.error-notice {
+	position: relative;
+	color: #d63638;
+	display: block;
+	margin: 0;
+	padding: 0;
+
+	@include breakpoint(sm) {
+		display: flex;
+		align-items: flex-start;
+		justify-content: left;
+	}
+
+	p {
+		margin: .25em 0;
+		color: $red_50;
+	}
+
+	pre {
+		background-color: #f1f1f1;
+		padding: 25px;
+		margin: 20px 0;
+		text-align: left;
+		white-space: pre-wrap;
+		color: #0f0f0f;
+		border-radius: 6px;
+	}
+}
+
+.icon {
+	color: inherit;
+	height: 20px;
+	width: 20px;
+	line-height: 0;
+}
+
+.offset {
+	padding-left: 0.5em;
+}
```

Comment:
Why is this necessary? It will cause UI differences when this component is later reused.